src/app/clusters/localization-configuration-server/localization-configuration-server.cpp

```diff
@@ -97,6 +97,8 @@ CHIP_ERROR LocalizationConfigurationAttrAccess::Read(const ConcreteReadAttribute
 // Pre-change callbacks for cluster attributes
 // =============================================================================
 
+using imcode = Protocols::InteractionModel::Status;
```

Comment:
```suggestion
using Status = Protocols::InteractionModel::Status;
```
is more consistent with how other code looks.